scripts/slowdown.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
I will use this to measure throughput in the different cluster configurations, to make sure that the penalty introduced by the transaction wrapper and shard table replication is less than the speedup introduced by reducing the number of synchronous replicas.